### ISSUE TITLE
onChange not working with onMouseUp/onTouchEnd on input type=range

### ISSUE THREAD

**Issue by akush** [preactjs/preact#1989]
For input type=range
```
<input
          id={`range-for-${uniqueId}`}
          className={getStyleClass(value)}
          onMouseUp={onDragEnd}
          onTouchEnd={onDragEnd}
          type="range"
          min="0"
          max="10"
          value={value}
          onChange={onChange}
          list="steplist"
        />
```
onChange is not working here. It was working on v10.beta-3 release and not on rc or stable v10.

**Comment by marvinhagemeister** [comment]
I think [onInput](https://preactjs.com/guide/v10/differences-to-react#use-oninput-instead-of-onchange) is the event you're looking for. We only patch `onChange` to `onInput` in `preact/compat` to get the React behaviour

**Comment by akush** [comment]
I am using `preact/compat` only. Still onChange is not working.

**Comment by JoviDeCroock** [comment]
Then we should see if it's related to https://github.com/preactjs/preact/pull/1760/files

**Comment by rmales** [comment]
I think this may actually be due to the fix for #1816. I think React does normalise onChange to onInput for range inputs (just not in IE11).
